src/core/server/licensing/license_feature.ts

```diff
@@ -0,0 +1,51 @@
+/*
+ * Licensed to Elasticsearch B.V. under one or more contributor
+ * license agreements. See the NOTICE file distributed with
+ * this work for additional information regarding copyright
+ * ownership. Elasticsearch B.V. licenses this file to you under
+ * the Apache License, Version 2.0 (the "License"); you may
+ * not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+import { LicensingServiceSetup } from './licensing_service_setup';
+import { LicenseFeatureSerializer } from './types';
+
+export class LicenseFeature {
+  private serializable: LicenseFeatureSerializer = service => ({
+    name: this.name,
+    isAvailable: this.isAvailable,
+    isEnabled: this.isEnabled,
```

Comment:
What's the difference between `isAvailable` and `isEnabled`?

Reply:
Not a clue, I'll see what I can find out. 😄